src/main/kotlin/com/example/coreweb/domains/contacts/models/entities/Contact.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.coreweb.domains.contacts.models.entities
+
+import com.example.coreweb.domains.base.entities.BaseEntity
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Table
+
+@Entity
+@Table(name = "contacts", schema = "core_web")
+class Contact : BaseEntity() {
+
+    @Column(nullable = false)
+    lateinit var name: String
+
+    @Column(nullable = false, unique = true)
+    lateinit var phone: String
+
+    var email: String? = null
+
+    var address: String? = null
```

Comment:
Address Should not be string.

Reply:
@sayemkcn vai then what will be the address field data type...?

Reply:
We need to create an address entity like `IntAddress` (International Address). It'll consist global address template. Let's sit tonight about it.

Reply:
Weight. I'll create an architecture to model how locations will work.

Reply:
@Mir00r please update the PR removing address field. We'll merge it, then after completing Location feature we'll add location there.